src/main/java/application/MenuController.java

```diff
@@ -7,10 +7,7 @@
 import beans.NewFormResponse;
 import beans.NotificationMessage;
 import beans.SessionNavigationBean;
-import beans.menus.BaseResponseBean;
-import beans.menus.EntityDetailListResponse;
-import beans.menus.EntityDetailResponse;
-import beans.menus.UpdateRequestBean;
+import beans.menus.*;
```

Comment:
This is slightly annoying, but Android Studio (and hence the `commcare-core` code) has been defaulting to never use `*` for imports, while IntelliJ will compress them like this. Recently I've switched IntelliJ to never compress as well to prevent unnecessary divergence in the `commcare-core` branch Can configure that in Preferences >Editor>Code Style>Java>Class count to use import with '*' set to 99 (see picture) 
<img width="626" alt="screen shot 2017-12-21 at 2 28 00 pm" src="https://user-images.githubusercontent.com/2293064/34271699-74936c38-e65b-11e7-91a9-7e1ad08137db.png">